sensor_msgs/Image image_left
sensor_msgs/Image image_right